examples/tutorial_1_connect.md

```diff
@@ -84,16 +84,27 @@ We can find out which port was attributed, and the resulting local addresses, by
 
 We'll **dial** the first switch from the second one, by specifying it's **Peer ID**, it's **MultiAddress** and the **`Ping` protocol codec**:
 ```nim
-  let conn = await switch2.dial(switch1.peerInfo.peerId, switch1.peerInfo.addrs, PingCodec)
+  let stream = await switch2.dial(switch1.peerInfo.peerId, switch1.peerInfo.addrs, PingCodec)
```

Comment:
What you get from the `switch.dial` might not necessarily be a `stream`, this is very protocol dependent and might be a real connection in some cases.

Reply:
Yes and no, currently all transports are multiplexed with mplex, however that doesn't have to be the case. For example, Quic will most likely bypass multiplexing at the libp2p level since it's natively multiplexed, same for WebRTC. The point is that, we cannot asume that we're getting a multiplexed stream here, even if the underlying transport is doing multiplexing, because it might actually not be the case. But maybe it makes sense to refer to this as `streams` at the high level libp2p api?

Reply:
Because we are instantiated our switch using `newStandardSwitch`, this will always give us a multiplexed tcp stream (based on the current codebase), no? Or is that something is meant to be changing in the future?

Reply:
Not really, there is no reason not to add other multiplexers or even disable the multiplexer altogether (QUIC), the point is mostly about nomenclature, so far `connection` and `stream` have been used interchangeably, and there is a good reason for that, i.e. you never know what you're getting as it's completely dependent on the transport and the multiplexer. It's a minor point and I'm not trying to make a big deal out of it, just give some perspective 👍